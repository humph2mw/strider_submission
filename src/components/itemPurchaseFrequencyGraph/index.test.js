import React from 'react'
import TestRenderer from 'react-test-renderer'
import ItemPurchaseFrequencyGraph from './index'

it('renders correctly', () => {
  const tree = TestRenderer
    .create(<ItemPurchaseFrequencyGraph />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
