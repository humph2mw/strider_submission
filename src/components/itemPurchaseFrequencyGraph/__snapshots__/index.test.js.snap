// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card"
  id="itemPurchaseFrequencyCard"
>
  <div
    className="card-title h5"
  >
    Item Order Frequency
  </div>
</div>
`;
